src/ducks/Alert/components/AlertRestrictionMessage/AlertRestrictionMessage.module.scss

```diff
@@ -0,0 +1,19 @@
+.wrapper {
+  min-height: 56px;
+  padding-left: 48px;
+  background-color: var(--texas-rose-light);
```

Comment:
`[D-0-1]` Use color variables defined by the [`san-webkit`](https://github.com/santiment/san-webkit/blob/master/src/styles/color.css#L37)